Cargo.toml

```diff
@@ -71,3 +71,6 @@ rustls = "0.19"
 tokio-rustls = "0.22"
 webpki = "0.21"
 webpki-roots = "0.21"
+
+[package.metadata.docs.rs]
+features = ["stream", "unstable"]
```

Comment:
Unstable too? It's really not meant to be used by anything except the companion `h2-tests` crate XD

Reply:
I removed `unstable`.